root.spec

```diff
@@ -22,6 +22,21 @@ Requires: freetype
 
 %define keep_archives true
 
+%define subpackageDebug true
+
+%if "%{?subpackageDebug:set}" == "set"
+%define envOptions USER_CXXFLAGS="-fdebug-prefix-map=%{cmsroot}=%{installroot} -fdebug-prefix-map=%{instroot}=%{installroot} -g -gdwarf-2 -O0"
+%endif
+# For non-linux archs disable subpackageDebug otherwise add dwz.file build dependencies.
+%if "%{?subpackageDebug:set}" == "set"
+%if %islinux
+BuildRequires: dwz file
```

Comment:
I looked at ROOT 6.04.XX debug builds we used together with Axel. RPM (with compression) was ~610M. Installed 1.7G. Thus make sure this is optimised as much as possible as we accidentally could run out of free space on architecture volumes (happens from time to time).